### ISSUE TITLE
Doubled single quotes does NOT act as double quotes

### ISSUE THREAD

**Issue by Explorer09** [onceupon/Bash-Oneliner#36]
Error in the README "Variable substitution within quotes" section.

```sh
# doubled single quotes act as double quotes making variables expand
echo ''$foo''
# bar
```

Technically it doesn't act as double quotes, but acts as if there were no quotes at all.
The difference lies on word splitting.
An example with `mkdir` can show the difference:

```sh
foo='a b'
mkdir $foo # Expected result: two directories, 'a' 'b'
mkdir ''$foo'' # Expected result: two directories, 'a' 'b'
mkdir "$foo" # Expected result: one directory, 'a b'
```

**Comment by onceupon** [comment]
agree, fix in [3e6bbf7](https://github.com/onceupon/Bash-Oneliner/commit/3e6bbf7c2036ab6d3bca1cc72c0648d68960ff67), thank you for the example as well.

**Comment by mu578** [comment]
```
# going on next level:
foo=("cat" "dog") && mkdir "${foo[@]}" && rmdir cat dog
```
